tests/acceptance/features/coreApiSharePublicLink1/accessToPublicLinkShare.feature

```diff
@@ -1,59 +0,0 @@
-@api @public_link_share-feature-required @files_sharing-app-required @issue-ocis-reva-282
-Feature: accessing a public link share
```

Comment:
will this features not implemented on ocis ?

Reply:
i had told the reason in the description. Please read that

Reply:
Yes i have read it, just wanted to know if it would never be implemented on ocis.

Reply:
in future, it can be implement not sure but it is sure that it will not implement like core